acceptance/setup/common/pre-suite/031_add_local_nix_user.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'bolt_setup_helper'
+
+test_name "Configure a bolt users on non-windows" do
```

Comment:
```suggestion
test_name "Configure a bolt user on nix" do
```

Reply:
Technically I do configuration for both the `local_user` and the `root` user (in the case of el-7). It did not make sense to me to create a whole new presuite file just to handle disabling requiretty for el7 root user. 

I can change non-windows to *nix.